contracts/stellar-interchain-token-service/src/contract.rs

```diff
@@ -439,6 +439,22 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
 
         Ok(())
     }
+
+    #[only_operator]
```

Comment:
dangerous to allow adding a minter for any existing token, that too from the operator and not the owner

Reply:
we shouldn't add functionality in production contracts just to facilitate testing. should handle this in a different manner